src/components/Settings/Notifications/NotificationsPushover/index.tsx

```diff
@@ -38,12 +38,18 @@ const NotificationsPushover: React.FC = () => {
   );
 
   const NotificationsPushoverSchema = Yup.object().shape({
-    accessToken: Yup.string().required(
-      intl.formatMessage(messages.validationAccessTokenRequired)
-    ),
-    userToken: Yup.string().required(
-      intl.formatMessage(messages.validationUserTokenRequired)
-    ),
+    accessToken: Yup.string()
+      .required(intl.formatMessage(messages.validationAccessTokenRequired))
+      .matches(
+        /^a[A-Za-z0-9]{29}$/,
```

Comment:
Are we absolutely certain this will always be 29 characters? I mean, do we really need strict validation like this here? This is on the user to enter the correct string, I feel.

Reply:
It is always 30 characters, the first of which is `a` (user/group tokens always begin with `u` or `g`).  I'm a Pushover user myself and this is clearly documented.

Reply:
Documentation:
https://pushover.net/api#registration
https://pushover.net/api#identifiers